src/ai-bundle/src/AiBundle.php

```diff
@@ -611,6 +612,9 @@ private function processAgentConfig(string $name, array $config, ContainerBuilde
                 ->setArguments([
                     $config['prompt']['text'],
                     $includeTools ? new Reference('ai.toolbox.'.$name) : null,
+                    TranslatorInterface::class,
+                    $config['prompt']['enable_translation'],
```

Comment:
Does it make sens to have both
- Translator
- enable_translation

in the constructor @chr-hertel ?

I would expect to just pass the translator with something like
```
$config['prompt']['enable_translation'] ? new Reference('translator') : null
```

And maybe if you use enable_translation = true, but have no registered translator we should throw an exception asking the user to install symfony/translator instead of a being "silent" about the error.

Reply:
i'd keep it separate, but you're right - we could get an error here.
i guess in that case we should throw an exception, that people need to install the translator. something like
```php
if (!interface_exists(TranslatorInterface::class)) {
    throw new RuntimeException('For using the translations with the prompt, the translator is required. Try running "composer require symfony/translation".');
}
```

Reply:
Yes this check is needed

Reply:
Correct me if I'm wrong, but isn't it always going to exist as it's a dependency of `symfony-ai/agent`?

Reply:
Can it be? It's imported into `SystemPromptInputProcessor` for type hinting the constructor.